docs/user/tutorial.rst

```diff
@@ -197,24 +219,92 @@ OK, now let's wire up this resource and see it in action. Go back to
 
     import falcon
 
-    import images
+    from .images import Resource
 
 
     api = application = falcon.API()
 
-    images = images.Resource()
+    images = Resource()
     api.add_route('/images', images)
 
 Now, when a request comes in for "/images", Falcon will call the
 responder on the images resource that corresponds to the requested
 HTTP method.
 
-Restart gunicorn, and then try sending a GET request to the resource:
+Restart Gunicorn, and then try sending a GET request to the resource:
 
 .. code:: bash
 
     $ http GET localhost:8000/images
 
+Testing your application
+------------------------
+
+Up to this point we didn't care about tests, but when creating applications
+that will be used by someone you should have those. So, as an exercise, we'll create
+the next piece of code in accordance with `Test Driven Development
+<http://www.obeythetestinggoat.com/book/praise.harry.html>`_
```

Comment:
Hmm... this one is the best I know, even though I'm not a fan of Django :) I'll add a note with explanations.